h5p-row.js

```diff
@@ -30,12 +30,31 @@ H5P.Row = (function (EventDispatcher) {
     var createHTML = function () {
       // Create wrapper
       wrapper = document.createElement('div');
+      wrapper.classList.add('h5p-row-wrapper');
 
-      // For now we just print the data :-)
-      const pre = document.createElement('pre');
-      pre.innerText = JSON.stringify(params, null, 2);
+      for (let i = 0; i < params.columns.length; i++) {
+        const column = document.createElement('div');
+        const columnData = params.columns[i];
+
+        column.classList.add('h5p-column');
```

Comment:
To match what H5P.Column does, add the class `h5p-row-content` here instead. H5P.Column itself will add the `h5p-column` class (but H5P.Row should not rely on this).

```suggestion
        column.classList.add('h5p-row-content');
```